app/views/posts/show.html.erb

```diff
@@ -23,6 +23,13 @@
                 <p>
                     <%= comment.author.name %> : <%= comment.text %>
                 </p>
+                <div>
+                    <% if can? :destroy, @post %>
+                      <%= form_with model: [@current_user, @post], method: :delete do |f| %>
+                        <%= f.submit "Delete" %>
+                      <% end %>
+                    <% end %>
+                </div>
```

Comment:
I think this button is responsible for deleting a post and it should not be in the comment section. So, I'm gonna move this button beside the Like, and Comment buttons and create another one for deleting a comment.

Reply:
@IndieCoderMM Okay, thank you for the feedback.